src/App.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Kindly avoid disabling linters. They are used for maintaining and writing clean code.